lib/zeromq/majordomo/asynchronous_arbiter.rb

```diff
@@ -0,0 +1,49 @@
+require 'ffi-rzmq'
+
+module Zeromq
+  module Majordomo
+    class AsynchronousArbiter
+      INFINITE = -1
+
+      class << self
```

Comment:
Well...the current Arbiter parent class assumes this is a singleton, doesn't it? I suppose it would be possible to mix this somehow but is it a good idea to mix the types? Or are you advocating that I switch it entirely (which I would be open to doing...)

Example: https://github.com/sittercity/arbiter/blob/develop/lib/arbiter.rb